### ISSUE TITLE
Extensions: outline over buttons not clearly visible

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#8622]
In light theme when I press tab to navigate to the buttons of an extension, I cannot clearly see that the button has focus.

Related https://github.com/Microsoft/vscode/issues/7966

**Comment by joaomoreno** [comment]
![image](https://cloud.githubusercontent.com/assets/22350/16558416/db1d07be-41e5-11e6-834c-a6a7d3346a2c.png)

Looks OK in my screen... it's hard to pick another blue since that is the one we use everywhere. We can maybe experiment with the background colour of the button itself.

**Comment by bpasero** [comment]
@joaomoreno actually I am referring to the feedback when an item is selected in blue. Then you end up with blue on blue.

**Comment by joaomoreno** [comment]
Made blue stronger in 74eacec3deba9a874f84d4262474eab544921b2f

**Comment by isidorn** [comment]
I think this works nice if the tree item has focus, but not if it is selected (dark blue). Then it is not visible that the button has a blue outline

**Comment by sandy081** [comment]
I had the same observation as Isi, also see the following video

![jul-28-2016 13-52-18]
